src/index.ts

```diff
@@ -114,8 +116,52 @@ export class EppoJSClient extends EppoClient {
    */
   public static initialized = false;
 
+  public static buildAndInit(config: IClientConfig): EppoJSClient {
+    const flagConfigurationStore =
+      config.flagConfigurationStore ??
+      configurationStorageFactory({
+        forceMemoryOnly: true,
+      });
+    const client = new EppoJSClient({ flagConfigurationStore });
+
+    // init will resolve the promise that client.waitForConfiguration returns.
+    client.init(config);
+    return client;
+  }
+
+  /**
+   * Resolved when the client is initialized
+   * @private
```

Comment:
super minor, but does this annotation really add anything given the `private` keyword two lines later?

Reply:
nope. eager IDE.